src/order/order.ts

```diff
@@ -60,3 +61,9 @@ export interface GiftCertificateOrderPayment extends OrderPayment {
         remaining: number;
     };
 }
+
+export type ProviderData =  SEPAProviderData;
```

Comment:
Are expecting other values to join this as a union?

What can be defined as `ProviderData`?